Source/WebKit/WebProcess/glib/WebProcessGLib.cpp

```diff
@@ -167,6 +165,12 @@ void WebProcess::platformInitializeWebProcess(WebProcessCreationParameters& para
     setDefaultCairoFontOptions(CairoUniquePtr<cairo_font_options_t>(cairo_font_options_create()));
 #endif
 
+    if (parameters.overrideDefaultCairoFontOptionsForTesting) {
+        CairoUniquePtr<cairo_font_options_t> fontOptions(cairo_font_options_create());
+        cairo_font_options_set_hint_style(fontOptions.get(), CAIRO_HINT_STYLE_NONE);
+        setDefaultCairoFontOptionsForTesting(WTFMove(fontOptions));
```

Comment:
Why don't we call `setDefaultCairoFontOptions() ` this will always be called after the main where the previous call is done.